### PR TITLE
feat(VM): reorganize stateful precompile docs

### PR DESCRIPTION
**Pull request by ashucoder9**
https://github.com/ava-labs/avalanche-docs/pull/1584#issue-1999391804

This PR aims to quickly resolve the build issues and merge conflicts arising from https://github.com/ava-labs/avalanche-docs/pull/1584

### CODE REVIEW

**Comment by ashucoder9** [comment]
Hey @meaghanfitzgerald, I can see that some of the VM docs are not translated into the Spanish version of the docs website. Example: https://docs.avax.network/es/build/vm/create/rust-vm and https://docs.avax.network/es/build/vm/evm/hello-world-precompile-tutorial

Is this something that is impacting the Translator bot?

**Comment by ashucoder9** [comment]
/gt docs/build/vm/evm/background-and-reqs.md i18n/es/docusaurus-plugin-content-docs/current/build/vm/evm/background-and-reqs.md spanish

**Comment by ashucoder9** [comment]
/gt docs/build/vm/evm/defining-precompile.md i18n/es/docusaurus-plugin-content-docs/current/build/vm/evm/defining-precompile.md spanish

**Comment by ashucoder9** [comment]
/gt docs/build/vm/evm/defining-tests.md i18n/es/docusaurus-plugin-content-docs/current/build/vm/evm/defining-tests.md spanish

File: i18n/es/docusaurus-plugin-content-docs/current/build/vm/evm/defining-precompile.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
+---
+etiquetas: [Construir, Máquinas Virtuales]
```
Comment (ashucoder9) [inline i18n/es/docusaurus-plugin-content-docs/current/build/vm/evm/defining-precompile.md:2]:
```suggestion
tags: [Construir, Máquinas Virtuales]
```

File: i18n/es/docusaurus-plugin-content-docs/current/build/vm/evm/generate-precompile.md [hunk 2]
```diff
@@ -0,0 +1,329 @@
+---
+etiquetas: [Construir, Máquinas Virtuales]
+descripción: Generando tu precompilado
```
Comment (ashucoder9) [inline i18n/es/docusaurus-plugin-content-docs/current/build/vm/evm/generate-precompile.md:3]:
```suggestion
description: Generando tu precompilado
```

File: i18n/es/docusaurus-plugin-content-docs/current/build/vm/evm/intro.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
+---
+etiquetas: [Construir, Máquinas Virtuales]
```
Comment (ashucoder9) [inline i18n/es/docusaurus-plugin-content-docs/current/build/vm/evm/intro.md:2]:
```suggestion
tags: [Construir, Máquinas Virtuales]
```

File: i18n/es/docusaurus-plugin-content-docs/current/build/vm/evm/intro.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+descripción: Guía para empezar a construir precompilaciones estatales para Subnet-EVM y Precompile-EVM
```
Comment (ashucoder9) [inline i18n/es/docusaurus-plugin-content-docs/current/build/vm/evm/intro.md:3]:
```suggestion
description: Guía para empezar a construir precompilaciones estatales para Subnet-EVM y Precompile-EVM
```

**Review by dubrado — DISMISSED** [review]

**Comment by laviniat1996** [comment]
@ashucoder9 can you please fix the failing stuff on this one?

**Pull request merged**